caddytls/certificates.go

```diff
@@ -182,6 +182,23 @@ func makeCertificate(certPEMBlock, keyPEMBlock []byte) (Certificate, error) {
 	if err != nil {
 		log.Printf("[WARNING] Stapling OCSP: %v", err)
 	}
+	// TODO: We don't have the Config.CTLogURLs here, just hardcoded some
+	// Config.Managed (with Let's Encrypt) specific values here.
+	// TODO: Venafi was just distrusted; one idea would be to fetch the
+	// Chrome list of trusted logs and just submit to everything until we
+	// have 1 Google and 1 non-Google. This would be hilarious, and pretty
+	// resillient.
+	sctLogURLs := []string{
+		"https://ct.googleapis.com/icarus",
+		"https://ctlog.api.venafi.com",
+	}
+	// TODO: We can skip this if the leaf has SCTs in an X.509 extension.
```

Comment:
Can this be done easily? Like, before merging? If not, let's do it ASAP after the release on Thursday.